containers/Home/index.ts

```diff
@@ -0,0 +1,9 @@
+import { About } from './About/About'
+import { Contact } from './Contact/Contact'
+import { Language } from './Language/Language'
+import { Lib } from './Lib/Lib'
+import { Skill } from './Skill/Skill'
+import { Achieve } from './Achieve/Achieve'
+import { Service } from './Service/Service'
+
+export { About, Contact, Language, Lib, Skill, Service, Achieve }
```

Comment:
This could be shorten by using this syntax for much cleaner. 

`export { About } from './About/About'`

The child component folder should have an `index.ts` file which contain the same purpose as this one, accept we don't  export the `styles.ts` (since its' not used anywhere else)